test/setup/make_source.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+echo "#!/usr/bin/env bash" > ../source.sh
+
+project_id=$(terraform output project_id)
+sa_json=$(terraform output sa_key)
+
+# shellcheck disable=SC2086,SC2154
+{ echo "export TF_VAR_project_id='$project_id'"; \
+echo "export TF_VAR_parent_resource_project='$project_id'"; \
+echo "export TF_VAR_parent_resource_folder='$TF_VAR_folder_id'"; \
```

Comment:
We should use outputs from Terraform to generate these values (mirror them through the Terraform config).

ie. in our outputs.tf add:
```
output "parent_resource_folder" {
  value = var.folder_id
}
```

Reply:
I don't like this way as we don't use the `parent` vars anywhere during the setup phase so those are really unnecessary pieces of HCL.
However fixed [here](https://github.com/terraform-google-modules/terraform-google-log-export/compare/d9bd46a326a17ba2bd80b9ef2fcd09ae049865e7..21c7079f7f4585d8304ce8988f0ec8ecffc2fdf8)